jdaviz/utils.py

```diff
@@ -1,20 +1,26 @@
 import os
 import time
 import threading
+import warnings
 from collections import deque
+from urllib.parse import urlparse
 
 import numpy as np
 from astropy.io import fits
 from astropy.utils import minversion
+from astropy.utils.data import download_file
 from astropy.wcs.wcsapi import BaseHighLevelWCS
+from astroquery.mast import Observations, Conf
```

Comment:
I think you actually want the lowercase `conf` (instantiated mast config). See https://github.com/astropy/astroquery/blob/25e6521ad3e86f5b2285d08e736cd3417159482d/astroquery/mast/__init__.py#L31

```suggestion
from astroquery.mast import Observations, conf
```